src/reflect/scala/reflect/internal/Scopes.scala

```diff
@@ -60,7 +62,7 @@ trait Scopes extends api.Scopes { self: SymbolTable =>
    */
   class Scope protected[Scopes]() extends AbstractIterable[Symbol] with ScopeApi with MemberScopeApi {
 
-    scopeCount += 1
+    scopeCount.incrementAndGet()
```

Comment:
Is this only used for statistics?

If yes then it should be masked
If no then there is a TOTOU somewhere